packages/client/src/components/atoms/BirthdayMenu/BirthdayMenu.tsx

```diff
@@ -73,9 +73,12 @@ const BirthdayMenu: React.FC<BirthdayMenuProps> = ({ customers }) => {
         badgeContent={getTodaysBirthdays}
         data-testid={__birthdayMenu__}
       >
-        <IconButton onClick={handleBirthdaysClick} size="large">
+        <Button
+          onClick={handleBirthdaysClick}
+          className="h-8 w-8 !p-0.5 ml-2  hover:bg-black/10"
```

Comment:
Loving the styling more/less. However, the user avatar below is 44px wide. It would be good to reflect that with the birthday menu button as items on the screen seem aligned that way. Secondly, when there's a badge, the button is almost not clickable.
Therefore, with a little playing around, I found this styling works nicely:
- width (and height) the same as user avatar: 44px
- padding: 8px (for the button to not appear too large and leave room for background to be shown on hover)
- leave the `ml-2` (works quite nicely)
- make the background white, instead of black on hover, as the background (of the header) is dark, opacity `10` works in that sense also (`hover:bg-white/10`)

Reply:
Also, the `Badge` component needs `select-none` and `cursor-normal` classes (otherwise it shows a weird text select cursor over that tiny badge)